sdk/search/tests.yml

```
@@ -4,6 +4,9 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: search
+    ${{ if contains(variables['Build.DefinitionName'], 'tests-weekly') }}:
+      MatrixReplace:
+        - ArmTemplateParameters=.*/@{ enableCmk = $true }
```

`=` and `\` characters in the query have to be escaped, see last bullet of [docs](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/job-matrix/README.md#replacemodifyappend-values).

```suggestion
        - ArmTemplateParameters=.*/@{ enableCmk \= $true }
```